### DISCUSSION TITLE
How to get the same results in file like on screen?

### DISCUSSION THREAD

**Discussion by vshlykov** [openai/whisper#1669] [Q&A] [answered]
I try to use this command:
whisper --language ru --model large -o ./result -- test.mp4
and get good result on screen. 

But when I try to use this code:

`model = whisper.load_model('large', device='cuda')
audio = "E:/whisper/test.mp4"
result = model.transcribe(audio, language='ru')

result_string = ""

output_file = "subtitles.srt"  

with open(output_file, mode='w', encoding='utf-8') as f:
    for item in result["segments"]:
        start_time = f"{int(item['start'] // 60):02}:{item['start'] % 60:.3f}"
        end_time = f"{int(item['end'] // 60):02}:{item['end'] % 60:.3f}"
        text = item['text']
        subtitle_line = f"[{start_time} --> {end_time}] {text}\n"
        f.write(subtitle_line)`
I get different result. I mean this result don't has punctuation, has different string lenght and different words. Why? How to get the same?

**Comment by glangford** [comment, marked as answer]
The default settings for the command line are different than for Python. See these discussions, hopefully this is the source of the problem.

* https://github.com/openai/whisper/discussions/177#discussioncomment-3766539
* https://github.com/openai/whisper/discussions/274
